style.css

```diff
@@ -0,0 +1,179 @@
+
+* {
+    box-sizing: border-box;
+    font-family: "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif; 
+    margin: 0;
+    padding: 0;
+}
+/* pictures as a background-image inside the grid: */
+.all-pictures {
+    display: grid;
+    grid-column-gap: 6px;
+    grid-row-gap: 5px;
+    grid-template-columns: 300px 300px 300px 363px;
```

Comment:
`SUGGESTION`

Consider deleting the `grid-template-columns` property to make use of the default value or change to a relative length value rather than absolute. 

For example `grid-template-columns: 25% 25% 25% 25%;` will make sure that the grid takes a total of 100% of its parent element and 25% for each column. See effect below:

![apple effect](https://user-images.githubusercontent.com/36057474/62437234-48455400-b73a-11e9-8870-2fa39ac25446.png)